packages/react-spinner/src/stories/SpinnerBestPractices.md

```diff
@@ -1,5 +1,3 @@
-## Best practices
-
 ### Do
 
-### Don't
+Add `tabIndex={-1}` to your Spinner to allow it to be picked up by screen readers
```

Comment:
Is this what we want from an accessibility standpoint?

Reply:
Yeah, because `Spinner` will most likely be used in tandem with other controls, we want to be able to allow it to be focusable so screen readers can detect it